content/development/coding-guidelines.md

```diff
@@ -79,9 +79,19 @@ Foo::Bar(const char* bar1, T bar2)
 	...
 }
 
-//  Empty functions converted to single line functions
+// Empty functions in class should be converted to inline functions
 
-Foo::~Foo() {}
+class Example
+{
+    void FooFunction() {}
```

Comment:
Even if it is in a single line, the alignment of the various columns should be preserved.
Also you had spaces instead of tabs for indentation.

```suggestion
		void			FooFunction() {}
```